tests/fdb/test_fdb_mac_move.py

```diff
@@ -0,0 +1,129 @@
+        normalized_level = "basic"
+    loop_times = LOOP_TIMES_LEVEL_MAP[normalized_level]
+
+    duthost = duthosts[rand_one_dut_hostname]
+    conf_facts = duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
+
+    # reinitialize data plane due to above changes on PTF interfaces
+    ptfadapter.reinit()
+
+    router_mac = duthost.facts['router_mac']
+
+    port_index_to_name = {v: k for k, v in conf_facts['port_index_map'].items()}
+
+    # Only take interfaces that are in ptf topology
+    ptf_ports_available_in_topo = ptfhost.host.options['variable_manager'].extra_vars.get("ifaces_map")
+    available_ports_idx = []
+    for idx, name in ptf_ports_available_in_topo.items():
+        if idx in port_index_to_name and conf_facts['PORT'][port_index_to_name[idx]].get('admin_status',
+                                                                                         'down') == 'up':
+            available_ports_idx.append(idx)
+
+    vlan_table = {}
+    interface_table = defaultdict(set)
+    config_portchannels = conf_facts.get('PORTCHANNEL', {})
+
+    # if DUT has more than one VLANs, use the first vlan
+    name = conf_facts['VLAN'].keys()[0]
+    vlan = conf_facts['VLAN'][name]
+    vlan_id = int(vlan['vlanid'])
+    vlan_table[vlan_id] = []
+
+    for ifname in conf_facts['VLAN_MEMBER'][name].keys():
+        if 'tagging_mode' not in conf_facts['VLAN_MEMBER'][name][ifname]:
+            continue
+        tagging_mode = conf_facts['VLAN_MEMBER'][name][ifname]['tagging_mode']
+        port_index = []
+        if ifname in config_portchannels:
+            for member in config_portchannels[ifname]['members']:
+                if conf_facts['port_index_map'][member] in available_ports_idx:
+                    port_index.append(conf_facts['port_index_map'][member])
+            if port_index:
+                interface_table[ifname].add(vlan_id)
+        elif conf_facts['port_index_map'][ifname] in available_ports_idx:
+            port_index.append(conf_facts['port_index_map'][ifname])
+            interface_table[ifname].add(vlan_id)
+        if port_index:
+            vlan_table[vlan_id].append({'port_index': port_index, 'tagging_mode': tagging_mode})
+
+    fdb = get_fdb_dict(duthost, ptfadapter, vlan_table)
+    port_list = fdb.keys()
+    dummy_mac_list = fdb.values()
+
+    for loop_time in range(0, loop_times):
+        port_index_start = (0 + loop_time) % len(port_list)
+
+        for (port, dummy_mac_set) in zip(range(len(port_list)), dummy_mac_list):
+            port_index = (port_index_start + port) % len(port_list)
+            for dummy_mac in dummy_mac_set:
+                send_arp_request(ptfadapter, port_index, dummy_mac, router_mac, vlan_id)
+
+        time.sleep(FDB_POPULATE_SLEEP_TIMEOUT)
+        time.sleep(10)
+        # Flush dataplane
+        ptfadapter.dataplane.flush()
+        fdb_cleanup(duthosts, rand_one_dut_hostname)
+        time.sleep(10)
```

Comment:
What is the purpose of this sleep? Can it be replaced with wait_until?

Reply:
The aim of this sleep is to stop for a few seconds when we finish one loop, and then start next loop. Maybe this sleep can not be replaced?

Reply:
Do you have guarantee that fdb cleanup is done properly? If not, would it cause next test to fail?

Reply:
Oh, the function `fdb_cleanup` will finally call the funtion `fdb_table_has_no_dynamic_macs` which use wait_until to make sure fdb entries will be cleaned up. When the fdb entries are not cleaned up, it will fail here.

Reply:
Please add a comment to state the intention.

Reply:
done